### DISCUSSION TITLE
EL RAT

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#313] [General]
Author: @AbbyANoble 
Reviewer: @kathypinzon 

Link: https://www.overleaf.com/read/pnqpwbfhxwvz#b7d538

**Comment by kathypinzon** [comment]
#5 I would say "which graph represents" instead of which of the answer choices.

**Comment by kathypinzon** [comment]
#7 there is a stray comma after answer choice c

**Comment by AbbyANoble** [comment]
Fixed and linked in #417
